pages/createForm/index.js

```diff
@@ -0,0 +1,28 @@
+import ActivityForm from "@/components/ActivityForm";
+import Link from "next/link";
+import styled from "styled-components";
+
+const StyledAddLink = styled(Link)`
+  background-color: orange;
+  text-decoration: none;
+  color: black;
+  border: 1px solid black;
+  padding: 5px;
+  border-radius: 5px;
+`;
+
+const StyledFieldset = styled.fieldset`
+  margin: 20px 30px 50px;
+`;
+
+export default function CreateForm({ onAddActivity }) {
```

Comment:
```suggestion
export default function CreateActivity({ onAddActivity }) {
```
I think this name fits better. You should also rename the file accordingly.